docs/reference/commandline/daemon.md

```diff
@@ -643,4 +644,20 @@ set like this:
     /usr/local/bin/docker daemon -D -g /var/lib/docker -H unix:// > /var/lib/docker-machine/docker.log 2>&1
 
 
+# Default cgroup parent
```

Comment:
@LK4D4  Is this value only valid when the `native.cgroupdriver` is `cgroupsfs`? Or does it work if that value is also `systemd`?